pkg/sippyserver/pr_commenting_processor.go

```diff
@@ -555,6 +543,59 @@ func (aw *AnalysisWorker) processRiskAnalysisComment(prPendingComment models.Pul
 	log.Debugf("Comment added to pendingComments: %s/%s/%s", pendingComment.org, pendingComment.repo, pendingComment.sha)
 }
 
+func buildComment(sortedAnalysis RiskAnalysisEntryList, sha string) string {
+	var sb strings.Builder
+	sb.WriteString(fmt.Sprintf("Job Failure Risk Analysis for sha: %s\n\n| Job Name | Failure Risk |\n|:---|:---|\n", sha))
+
+	for _, value := range sortedAnalysis {
+		tableKey := value.Key
+
+		if value.Value.URL != "" {
+			tableKey = fmt.Sprintf("[%s](%s)", value.Key, value.Value.URL)
+		}
+
+		var riskSb strings.Builder
+		riskSb.WriteString(fmt.Sprintf("<b>%s</b>", value.Value.RiskLevel.Name))
```

Comment:
Is the html style required in a github table? Just curious if we could do markdown syntax for all these tags.

I'd love to see this made a little larger so we could also highlight the test name somehow.